api/src/tests/accounts_test.rs

```diff
@@ -150,25 +153,40 @@ async fn test_get_account_resources_by_invalid_ledger_version() {
     context.check_golden_output(resp);
 }
 
-// figure out a working module code, no idea where the existing one comes from
-#[ignore] // TODO(issue 81): re-enable after cleaning up the compiled code in the test
 #[tokio::test(flavor = "multi_thread", worker_threads = 2)]
 async fn test_get_account_modules_by_ledger_version() {
     let mut context = new_test_context(current_function_name!());
-    let code = "a11ceb0b0300000006010002030205050703070a0c0816100c260900000001000100000102084d794d6f64756c650269640000000000000000000000000b1e55ed00010000000231010200";
-    let root_account = context.root_account().await;
-    let txn = root_account.sign_with_transaction_builder(
-        context
-            .transaction_factory()
-            .module(hex::decode(code).unwrap()),
+
+    let mut builder = PackageBuilder::new("test_package");
+    builder.add_source(
+        "test_module.move",
+        r#"
+        module 0xa550c18::test_module {}
+        "#,
     );
+    let path = builder.write_to_temp().unwrap();
+
+    let package = BuiltPackage::build(path.path().to_path_buf(), BuildOptions::default())
+        .expect("Should be able to build a package");
+    let code = package.extract_code();
+    let metadata = package
+        .extract_metadata()
+        .expect("Should be able to extract metadata");
+    let payload = aptos_stdlib::code_publish_package_txn(
+        bcs::to_bytes(&metadata).expect("Should be able to serialize metadata"),
+        code,
+    );
+
```

Comment:
I wonder if we should add this code to https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/cached-packages/src/aptos_stdlib.rs

Reply:
Added